src/main/java/org/gridsuite/modification/server/entities/equipment/creation/TwoWindingsTransformerCreationEntity.java

```diff
@@ -40,24 +42,109 @@ public class TwoWindingsTransformerCreationEntity extends BranchCreationEntity {
     @Column(name = "ratedVoltage2")
     private double ratedVoltage2;
 
+    @Column(name = "rateds")
+    private Double ratedS;
+
+    @Column(name = "phasetapchangerlowtapposition")
+    private Integer phaseTapChangerLowTapPosition;
+
+    @Column(name = "phasetapchangertapposition")
+    private Integer phaseTapChangerTapPosition;
+
+    @Column(name = "phasetapchangerregulating")
+    private Boolean phaseTapChangerRegulating;
+
+    @Column(name = "phasetapchangertargetdeadband")
+    private Double phaseTapChangerTargetDeadband;
+
+    @Column(name = "phasetapchangerterminalrefconnectableid")
+    private String phaseTapChangerTerminalRefConnectableId;
+
+    @Column(name = "phasetapchangerterminalrefvoltagelevelid")
+    private String phaseTapChangerTerminalRefVoltageLevelId;
+
+    @Column(name = "phasetapchangerterminalreftype")
+    private String phaseTapChangerTerminalRefType;
+
+    @Column(name = "phasetapchangerregulationmode")
+    @Enumerated(EnumType.STRING)
+    private PhaseTapChanger.RegulationMode phaseTapChangerRegulationMode;
+
+    @Column(name = "phasetapchangerregulationvalue")
+    private Double phaseTapChangerRegulationValue;
+
+    @Column(name = "ratiotapchangerlowtapposition")
+    private Integer ratioTapChangerLowTapPosition;
+
+    @Column(name = "ratiotapchangertapposition")
+    private Integer ratioTapChangerTapPosition;
+
+    @Column(name = "ratiotapchangerregulating")
+    private Boolean ratioTapChangerRegulating;
+
+    @Column(name = "ratiotapchangertargetdeadband")
+    private Double ratioTapChangerTargetDeadband;
+
+    @Column(name = "ratiotapchangerterminalrefconnectableid")
+    private String ratioTapChangerTerminalRefConnectableId;
+
+    @Column(name = "ratiotapchangerterminalrefvoltagelevelid")
+    private String ratioTapChangerTerminalRefVoltageLevelId;
+
+    @Column(name = "ratiotapchangerterminalreftype")
+    private String ratioTapChangerTerminalRefType;
+
+    @Column(name = "ratiotapchangerloadtapchangingcapabilities")
+    private Boolean ratioTapChangerLoadTapChangingCapabilities;
+
+    @Column(name = "ratiotapchangertargetv")
+    private Double ratioTapChangerTargetV;
+
+    @ElementCollection
+    @CollectionTable(
```

Comment:
No lazy loading ?

Reply:
According to this source : https://en.wikibooks.org/wiki/Java_Persistence/ElementCollection#Basic_Collections
@ElementCollection annotation uses LAZY by default